Crashlytics/Crashlytics/Controllers/FIRCLSRolloutsPersistenceManager.m

```diff
@@ -57,7 +57,7 @@ - (void)updateRolloutsStateToPersistenceWithRollouts:(NSData *_Nonnull)rollouts
 
   NSFileHandle *rolloutsFile = [NSFileHandle fileHandleForUpdatingAtPath:rolloutsPath];
 
-  dispatch_sync(FIRCLSGetLoggingQueue(), ^{
+  dispatch_async(FIRCLSGetLoggingQueue(), ^{
```

Comment:
This solution works. But I think we might have to move to dispatch groups to make sure we have fine grained control of different tasks submitted. Moving it to async is going to make it the action performed to slow down. Is that expected?

Reply:
If this job has to be synchronous, then I would even recommend moving to a separate serial queue.

Reply:
I'm ok with it slowing down, because we're not waiting on the result. This queue is the one we use for most writes, so this will keep it serial with other writes in the SDK.